pandas/_libs/tslib.pyx

```diff
@@ -821,7 +821,7 @@ cdef _array_to_datetime_object(
                     oresult[i] = <object>NaT
                     continue
                 if is_raise:
-                    raise
+                    raise ValueError(f"Unable to parse string \"{val}\" at position {i}")
```

Comment:
thanks @mroeschke, will correct this.
`raise type(err)(f"Unable to parse string \"{val}\" at position {i}")` should correct this ( let me know if otherwise, apologies if it's incorrect/trivial question )